index.html

```diff
@@ -1,12 +1,348 @@
+          Товар, не дом, конечно же.</p>
+      </div>
+
+      <div class="service services-description-credit service-show">
+        <h2 class="service-title">Кредит</h2>
+        <p>Залезть в долговую яму стало проще! Кредитные консультанты подберут для вас наиболее выгодные условия
+          кредита. Выгодные для банка, разумеется.</p>
+      </div>
+    </section>
+
+
+    <section class="delivery">
+      <a class="delivery-link" href="#">
+        <img class="delivery-image" src="" alt="Логотип доставки">
+        <span class="delivery-name">Доставим редкий товар под заказ</span>
+      </a>
+    </section>
+
+    <section class="about-company">
+      <h2 class="visually-hidden">О компании и контактная информация</h2>
+      <section class="about-us">
+        <h2>О нас</h2>
+        <p>Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.
+        </p>
+        <p>Мы можем доставить ваш товар в самые отдалённые уголки России транспортными компаниями:
+        </p>
+        <ul class="about-us-list">
+          <li class="about-us-item">
+            <b>Меловые линии</b>
+          </li>
+          <li class="about-us-item">
+            <b>СДИК</b>
+          </li>
+          <li class="about-us-item">
+            <b>ТоксБерри</b>
+          </li>
+        </ul>
+        <a class="about-us-list button" href="#">Подробнее о нас</a>
+      </section>
+
+      <section class="contact">
+        <h2 class="contact-list">Контакты</h2>
+        <p>Вы можете забрать товар сами, заехав в наш офис.
+          Заодно сможете проверить работоспособность покупки.</p>
+        <h3 class="contact-address">Интернет-магазин «Девайс»</h3>
+        <address class="contact-address">
+          <p class="about-text">Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
+          <a class="contact-adress-phone" href="tel:+78128121212">+7 (812) 812-12-12</a>
+        </address>
+        <h3 class="contact-subtitle">Время работы</h3>
+        <ul class="contacts-schedule">
+          <li class="contacts-schedule-item">пн—пт: с 10:00 до 22:00</li>
+          <li class="contacts-schedule-item">сб—вс: с 10:00 до 19:00</li>
+        </ul>
+        <a class="contact button" href="#">Напишите нам</a>
+      </section>
+    </section>
+
+    <section class="subscribe">
+      <h2 class="subscribe">Подпишитесь
+        на рассылку</h2>
+      <p class="subscribe-description">Еженедельный дайджест с обзорами новинок,
+        <br>длительными тест-драйвами, распаковками, а также акции,<br> спецпредложения и вот это вот всё.
+      </p>
+      <h2 class="visually-hidden">Подписаться на рассылку</h2>
+      <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+        <input class="newsletter-email" placeholder="Ваш e-mail" type="email">
+        <button class="newsletter-button button" type="submit">Подписаться</button>
+      </form>
+      <p name="subscribe-free">Это бесплатно</p>
```

Comment:
```suggestion
      <p class="subscribe-free">Это бесплатно</p>
```
У тега `p` не может быть атрибута `name`.